storio-common/src/test/java/com/pushtorefresh/storio/internal/QueriesTest.java

```diff
@@ -110,4 +210,44 @@ public void nonEmptyListOfStringsToArray() {
             assertEquals(list.get(i), array[i]);
         }
     }
+
+    //endregion
+
+    //region Tests for Queries.nonNullString()
+
+    @Test
+    public void nonNullStringFromNull() {
+        assertEquals("", Queries.nonNullString(null));
+    }
+
+    @Test
+    public void nonNullStringFromEmptyString() {
+        assertEquals("", Queries.nonNullString(""));
+    }
+
+    @Test
+    public void nonNullStringFromNormalString() {
+        assertEquals("123", Queries.nonNullString("123"));
+    }
+
+    //endregion
+
+    //region Tests for Queries.nullableString()
+
+    @Test
+    public void nullableStringFromNull() {
+        assertEquals(null, Queries.nullableString(null));
```

Comment:
Wow, sure :)

I am thinking about AssertJ instead of JUnit, so it'll be replaced with
assertThat(someThing).isNull()
5 июня 2015 г. 1:59 AM пользователь "Dmitrii Nikitin" <
notifications@github.com> написал:

> In
> storio-common/src/test/java/com/pushtorefresh/storio/internal/QueriesTest.java
> <https://github.com/pushtorefresh/storio/pull/362#discussion_r31774604>:
>
> > +    public void nonNullStringFromEmptyString() {
> > +        assertEquals("", Queries.nonNullString(""));
> > +    }
> > +
> > +    @Test
> > +    public void nonNullStringFromNormalString() {
> > +        assertEquals("123", Queries.nonNullString("123"));
> > +    }
> > +
> > +    //endregion
> > +
> > +    //region Tests for Queries.nullableString()
> > +
> > +    @Test
> > +    public void nullableStringFromNull() {
> > +        assertEquals(null, Queries.nullableString(null));
>
> Maybe just assertNull?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pushtorefresh/storio/pull/362/files#r31774604>.
>